examples/crm/src/markdown.rs

```diff
@@ -0,0 +1,104 @@
+/// Original author of this code is [Nathan Ringo](https://github.com/remexre)
```

Comment:
don't you think we should move this into a separate crate?

Reply:
While I think this functionality is worth of it's own crate. I also believe that having it as part of the examples would make it easier for new people to the library (like me) to understand how to generate new dom elements that can be used within the html! macro.

Reply:
that's a good point! It can be made it's own crate separately.